examples/app/polymer/polymer.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3>Bound data</h3>
+    </div>
+    <div class="panel-body">
+        <pre>{{vm.formattedData()}}</pre>
+    </div>
+</div>
+
+<jsonforms data="vm.data" ui-schema="vm.uiSchema" schema="vm.schema"></jsonforms>
```

Comment:
Yes, the latter, vm is commonly used for ViewModel, but of course you are free to use whatever you like.